libraries/editor/src/lib/translations/af.js

```diff
@@ -0,0 +1,32 @@
+!(function (e) {
+    const i = (e.af = e.af || {});
+    (i.dictionary = Object.assign(i.dictionary || {}, {
+        'Align center': 'Belyn in die middel',
+        'Align left': 'Belyn links',
+        'Align right': 'Belyn regs',
```

Comment:
Something to consider: we could save a bit of space by just not including the translation files for CKEditor. If not on the site, then at least in the repo =P